asset.go

```diff
@@ -416,6 +422,9 @@ type ChainConfig struct {
 	XDti             string `yaml:"dti,omitempty"`
 	XCoinGeckoId     string `yaml:"coingecko_id,omitempty"`
 	XCoinMarketCapId string `yaml:"coinmarketcap_id,omitempty"`
+
+	// Number of seconds for which the transaction will be considered valid after submission. Used only by Stellar so far
+	TransactionValidityTime int `yaml:"transaction_validity_time"`
```

Comment:
use `time.Duration`, and then update configuration to use a string.  I think we can do much longer than 30s, let's say 2 hours (`2h`).

Reply:
otherwise simple things like gas fees changing, or temporary network connectivity issues will force people to have to re-sign, which can be annoying.  Better to allow a generous amount of time for the caller to resubmit in a robust way.